lib/health_checks/rails.rb

```diff
@@ -1,42 +1,71 @@
 require 'okcomputer'
-require 'health_checks/memory'
+require 'health_checks/custom/memory_check'
+require 'health_checks/custom/mongoid_check'
+require 'health_checks/custom/redis_check'
 
 module HealthChecks
   module_function
 
-  def rails
-    OkComputer::Registry.register 'database', OkComputer::MongoidCheck.new
-    OkComputer::Registry.register 'redis', RedisCheck.new
+  def rails(mongo_databases, redis_configs)
+    mongo_databases.each do |db|
+      mongoid_check = Custom::MongoidCheck.new(db)
+      OkComputer::Registry.register "mongoid #{db[:name]}", MongoidCheck.new(mongoid_check)
+    end
+
+    redis_configs.each do |config|
+      redis_check = Custom::RedisCheck.new(config)
+      OkComputer::Registry.register 'redis', RedisCheck.new(redis_check)
+    end
+
     OkComputer::Registry.register 'memory', MemoryCheck.new
```

Comment:
How is this working without being prefixed by `Custom::`?

Reply:
because this MemoryCheck is the check inheriting from OkComputer::Check and it's different from Custom::MemoryCheck. I couldn't come up with better names, so I decided that the Namespace will be enough distinguishing for both classes

Reply:
I think it would be better to distinguish these OKComputer checks either by naming them differently or by also namespacing them